server/index.js

```diff
@@ -0,0 +1,74 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+const db = require('../database/index.js');
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+app.use(express.static(__dirname + '/../dist'));
+
+const PORT = 3101;
+
+// This version is only used for dev testing
+app.get('/tiersIncluded', (req, res) => {
+  const dataString = { bundleId: 1 };
+  return db.tiersRequest(dataString)
+  .then(tiersData => {
+    if (tiersData) {
+      return tiersData;
+    } else {
+      res.status(404).send('no such bundleId');
+    }
+  })
+  .then(tiersData => {
+    if (tiersData) {
+      res.send(JSON.stringify(tiersData));
+    }
```

Comment:
Because you have the previous conditional, you are already checking to see if your data exists. Checking to see if there is a specific reason that you are not formatting and sending the data from there instead of here.